operator/patroni/patroni.go

```diff
@@ -0,0 +1,225 @@
+package patroni
+
+/*
+ Copyright 2020 Crunchy Data Solutions, Inc.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+import (
+	"errors"
+	"fmt"
+
+	log "github.com/sirupsen/logrus"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+
+	"github.com/crunchydata/postgres-operator/config"
+	"github.com/crunchydata/postgres-operator/kubeapi"
+)
+
+// dbContainerName is the name of the container containing the PG database in a PG (primary or
+// replica) pod
+const dbContainerName = "database"
```

Comment:
This probably needs to be a bit more of a global constant, knowing that I wrote some code that references this and like am doing a poor job at it.

Also does not need to be addressed now.

Reply:
Per our conversation, will leave as is for now, knowing constants in general need an overhaul and consolidation.

In the meantime I left a note on our reorg doc to get a conversation going around this.